doc/api/toolkits/mplot3d/view_angles.rst

```diff
@@ -28,11 +28,33 @@ as well as roll, and all three angles can be set programmatically::
     ax = plt.figure().add_subplot(projection='3d')
     ax.view_init(elev=30, azim=45, roll=15)
 
+Rotation of the plot
+====================
+
+The *azim*, *elev*, *roll* rotation order corresponds to rotation of the scene
+observed by a stationary camera. First, a left-handed rotation about the z axis is
+applied (*azim*), then a right-handed rotation about the (camera) y axis (*elev*), then a
+right-handed rotation about the (camera) x axis (*roll*). Here, the z, y, and x axis are fixed
+axes (not the axes that rotate together with the original scene).
+
+This can also be thought of as orbiting a camera around a fixed scene, by reversing
+the order of operations. First the camera is rotated about the scene's +x axis
+(*roll*), then the +y axis (*elev*), then the −z axis (*azim*).
+
+If you would like to make the connection with quaternions (because
```

Comment:
I considered it, but I get the impression that  `..note::` is not meant for the purpose of adding some reference material which is a bit secondary, as it seems is our intention; rather, it is for [information you want the user to pay particular attention to](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html). I wouldn't want to put that much emphasis on the particulars of the definition of the view angles.

Since the text already mentions that '[The view angles] are further documented in the `.mplot3d.axes3d.Axes3D.view_init` API', I thought it be sensible to move the information there (and keep the view_angles.rst as it was). I'll give it a try.